apt/map.jinja

```diff
@@ -21,18 +20,68 @@
         'default_url': 'http://deb.debian.org/debian/',
         'default_keyring_package': 'debian-archive-keyring',
         'repositories': {
-          'sane_default': {
+          'default': {
```

Comment:
@jdkelleher Although I prefer the names you set more than the old ones, changing these will be backward incompatible for users of this formula in the case that the formula was used *as-is* (with no pillar data): the existing repos `sane_default-binary.list` and `security-stable-binary.list` will still exist and the new ones (`default-binary.list` and  `default-security-binary.list`) will be created, giving a warning that the entries are duplicated.

I'd suggest you *keep the old* names for the Debian part, so your PR is backward compatible and can be merged. We can later submit another PR to rename these and allow to delete specific entries (which is not currently supported).

wdyt?

Reply:
Makes perfect sense to me.  I was thinking about the entry deletion gap when I was making updates, but moved forward without a solution.

Reply:
@javierbertoli I just committed the requested updates.  Please let me know if anything else is needed.  Thanks.